client/plots/hierCluster.config.js

```diff
@@ -57,8 +57,13 @@ export async function getPlotConfig(opts = {}, app) {
 			const tw = i.term ? i : { term: i }
 
 			if (!tw.term.type) {
-				// must provide term type for hierCluster
-				throw `term type is missing`
+				if (config.dataType) {
```

Comment:
config.dataType should be required and this should be verified elsewhere. also please define supported values for dataType. in case of geneExpression and metabolite dataType can equal to term type. however it's not the case for dictionary terms. thus here will need some if(dataType=='geneExpression') then auto assign